Scripts/datahandling/zonedata.py

```diff
@@ -64,26 +65,10 @@ def _add_transformations(self, data):
             0.5, self.zone_numbers, dtype=numpy.float32)
         self.share["share_male"] = pandas.Series(
             0.5, self.zone_numbers, dtype=numpy.float32)
-        share_7_99 = pandas.Series(0, self.zone_numbers, dtype=numpy.float32)
-        pop = data["population"]
-        wp = data["workplaces"]
-        hh = data["households"]
         for col in data:
-            if col.startswith("sh_age"):
-                pop_share = data[col]
-                self.share[col] = pop_share
-                col_name = col.replace("sh_", "")
-                self[col_name] = pop_share * pop
-                self[col_name + "_female"] = 0.5 * pop_share * pop
-                self[col_name + "_male"] = 0.5 * pop_share * pop
-                share_7_99 += pop_share
-            if col.startswith("sh_income"):
-                self[col.replace("sh_", "")] = data[col] * pop
-            if col.startswith("sh_wrk_"):
-                self[col.replace("sh_wrk_", "")] = data[col] * wp
-            if col.startswith("sh_hh"):
-                self[col.replace("sh_", "")] = data[col] * hh
-        self.share["share_age_7-99"] = share_7_99
+            if col.startswith("age"):
+                self[col + "_female"] = 0.5 * data[col]
```

Comment:
If I'm correct these variables are not used anymore. They were part of long distance generation, but removed from latest versions.